build-support/flake-repro/main.go

```diff
@@ -0,0 +1,180 @@
+package main
```

Comment:
This really should be a bash script and not Go source as all that is done is to shell out and run commands. It would be much easier to follow (and probably less code) if it were done in a shell scripting language like bash and would be consistent with the other build scripts. If you want some pointers on this ping me.